exporter/importables.go

```diff
@@ -561,6 +563,43 @@ var resourcesMap map[string]importable = map[string]importable{
 			case "url", "format":
 				return true
 			}
+			var js jobs.JobSettings
+			common.DataToStructPointer(d, ic.Resources["databricks_job"].Schema, &js)
+			switch pathString {
+			case "email_notifications":
+				if js.EmailNotifications != nil {
+					return reflect.DeepEqual(*js.EmailNotifications, jobs.EmailNotifications{})
+				}
+			case "webhook_notifications":
+				if js.WebhookNotifications != nil {
+					return reflect.DeepEqual(*js.WebhookNotifications, jobs.WebhookNotifications{})
+				}
+			case "notification_settings":
+				if js.NotificationSettings != nil {
+					return reflect.DeepEqual(*js.NotificationSettings, sdk_jobs.JobNotificationSettings{})
+				}
+			}
+			if strings.HasPrefix(pathString, "task.") {
+				parts := strings.Split(pathString, ".")
+				if len(parts) > 2 {
+					taskIndex, err := strconv.Atoi(parts[1])
+					if err == nil && taskIndex >= 0 && taskIndex < len(js.Tasks) {
+						switch parts[len(parts)-1] {
```

Comment:
what is `parts[len(parts)-1]`? Might be good to define it in variable so it's more code readable.

Reply:
`parts[len(parts)-1]` - block name

Reply:
I think doing something like would be more readable:
`blockName := parts[len(parts)-1]`